debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -166,14 +159,29 @@ public static String allOnlineLogsQuery() {
      * @return query
      */
     public static String archiveLogsQuery(Scn scn, Duration archiveLogRetention) {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("SELECT NAME AS FILE_NAME, NEXT_CHANGE# AS NEXT_CHANGE, FIRST_CHANGE# AS FIRST_CHANGE ");
+        sb.append("FROM ").append(ARCHIVED_LOG_VIEW).append(" ");
+        sb.append("WHERE NAME IS NOT NULL ");
```

Comment:
@Naros LGTM. One question, why did you change the predicate order in this query? Does it have any impact on execution plan?

Reply:
I changed it simply for readability mostly.  It had no impact that I've saw on execution plan.